app/presenters/gobierto_data/dataset_presenter.rb

```diff
@@ -0,0 +1,76 @@
+module GobiertoData
+  class DatasetPresenter
+    include ActionView::Helpers::UrlHelper
+
+    attr_reader :site
+
+    def initialize(site)
+      @site = site
+    end
+
+    def build_catalog
+      catalog = {
+        identifier_uri: url_helpers.gobierto_data_root_url(host: site.domain),
+        title:          "dcat catalog for #{site}",
+        description:    "this is catalog published by #{@catalog_identifier_uri} which contains datasets",
+        issued:         site.created_at,
+        modified:       GobiertoData::Dataset.where(site_id: site.id).maximum(:created_at) || site.created_at,
+        languages:      site["configuration_data"]["available_locales"],
```

Comment:
`site["configuration_data"]["available_locales"]` returns an array but the hash would exepect a String, so you might need to join the array and separate it with commas?

Reply:
mmmm I am using to print multiples `xml elements languages` (one xml element for every lang available) 
```
<%= @catalog[:languages].map { |locale| "<dct:language>#{locale}</dct:language>" }.join.html_safe %>
```

but searching into documentation and looks like this I'm wrong... only appear *single line of language* like:
```
<dct:language>en</dct:languague>
```
So I will replace with `site_locale`